github/pulls.go

```diff
@@ -134,6 +135,29 @@ func (s *PullRequestsService) Get(owner string, repo string, number int) (*PullR
 	return pull, resp, err
 }
 
+// Get pull request raw format (diff or patch)
+func (s *PullRequestsService) GetRaw(owner string, repo string, number int, opt *MediaRawTypeOption) (string, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/pulls/%d", owner, repo, number)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return "", nil, err
+	}
+
+	if opt.Type == Diff {
```

Comment:
`opt` is a pointer, but you never check if its value is nil.

Usually a pointer is used when `nil` means using default options. Either do that, or make it a value.

Making it a value makes more sense, since I can't think of a good "default" value for `MediaRawTypeOption`.